StrictEmit/ILGeneratorExtensions.Calls.cs

```diff
@@ -268,17 +268,35 @@ public static void EmitCallVirtual([NotNull] this ILGenerator il, [NotNull] Cons
         /// <summary>
         /// Attempts to cast an object passed by reference to the specified class.
         /// </summary>
+        /// <typeparam name="T">The type to cast the highest item on the stack to.</typeparam>
```

Comment:
This should be "on top of the stack" to use proper stack lingo. Same for the others.